scripts/remove-old-image.py

```diff
@@ -0,0 +1,82 @@
+"""
+
+This script is based on this documentation: 
+https://docs.github.com/en/rest/packages/packages?apiVersion=2022-11-28
+
+"""
+
+import os
+import requests
+from requests.auth import HTTPBasicAuth
+
+# Delete the old container (based on the previous tag)
+def delete_old_image(version_id, org, headers, auth):
+    url_delete_previous_version = f"https://api.github.com/orgs/{org}/packages/container/{container_name}/versions/{version_id}"
+    response = requests.delete(url_delete_previous_version, headers=headers, auth=auth)
+    if response.status_code == 204:
+        print(f'Previous container deleted!')
+    else:
+        print('error deleting the previous container:', response.status_code, response.text)
+        exit(1)
+    
+"""
+Find the previous tag for a specific container.
+Delete the previous tag if it exists and it is not the current one. This wont delete the current tag or older PR tags.
+Check if tags is not empty and check if the len is == 1. If that is the case, it means that the only tag is the previous one.
+"""
+def find_previous_container_tag(response, pr_tag):
```

Comment:
also want to see some unit tests suite for your functions

Reply:
Ok

Reply:
start with unit tests, they are cheaper and easier to test a variety of failure and exception cases

Reply:
Ok

Reply:
I will make this request in another issue. That being said, it is not related to this pull request as it requires more time. Check this issue (https://github.com/ai-cfia/devops/issues/26)